chainer/backends/cuda.py

```diff
@@ -66,7 +66,10 @@
     from cupy.cuda import Stream  # type: ignore # NOQA
 
     # Alias for ignoring the warning in setup.cfg
-    from cupy.util import PerformanceWarning as _PerformanceWarning  # NOQA
+    try:
+        from cupy._util import PerformanceWarning as _PerformanceWarning  # NOQA
+    except Exception as e:
```

Comment:
```

Reply:
I agree.  While `import cupy` may raise other exception, the exception class here is probably `ImportError`.